build/cli.js

```diff
@@ -184,6 +191,20 @@ async function buildDocuments() {
   return { slugPerLocale: docPerLocale, peakHeapBytes };
 }
 
+async function buildOtherSPAs() {
+  const outPath = path.join(BUILD_OUT_ROOT, "en-us", "_spas");
```

Comment:
I'm not convinced by this name. Perhaps it should just be `path.join(BUILD_OUT_ROOT, "en-us")` and the file we thus create is `client/build/en-us/404.html`. 
The reasons I wanted a folder was the foresight that we're going to build more of these server-side rendered non-document HTML files. E.g. home-page, and site-search, and sign-up. etc. 
But a page like `site-search` needs to become `client/build/en-us/search/index.html` (which the deployer uploads as `Key=en-us/search`).